src/CalcManager/Command.h

```diff
@@ -151,6 +152,34 @@ namespace CalculationManager
         CommandINV = 146,
         CommandSET_RESULT = 147,
 
+        CommandSEC = 400,
```

Comment:
Can you make sure these new commands are hooked up to the ButtonUsage diagnostics?

Reply:
These commands will not hook up to diagnostics until the UI portion of this changes comes in since the diagnostics is hooked up to the actual UI button rather than the calc engine commands.